installer/api.py

```diff
@@ -155,15 +149,41 @@ def __getJSON(retCode: int=0) -> Optional[Dict]:
 			"java": data.get(CMAKE_JAVA),
 			"hdf5": data.get(CMAKE_HDF5),
 			"jpeg": data.get(CMAKE_JPEG)
+		}
+	else:
+		version = {
+			"os": jsonData[0],
+			"architecture": jsonData[1],
+			"cuda": '',
+			"cmake": '',
+			"gcc": '',
+			"gpp": '',
+			"mpi": '',
+			"python": '',
+			"sqlite": '',
+			"java": '',
+			"hdf5": '',
+			"jpeg": ''
+		}
```

Comment:
Unnecessarily complex. It's easier to make the `exists` check inside function `parseCmakeVersions` and return an empty dict if not exists (before the open). Then, this function does not need to change, as the values will be set to `None` by the `get` function, and then converted tu `null` by `json.dumps`, which is the ideal value for unkown.